CODEOWNERS

```diff
@@ -95,6 +95,7 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/mcuboot/                         @nvlsianpu
 /modules/cjson/                           @simensrostad @plskeggs @sigvartmh
+/samples/                                 @nrfconnect/ncs-test-leads
```

Comment:
I guess something went wrong with rebasing? This commit doesn't seem to belong to this PR.

Reply:
Hi Robert,
I'm still trying to fix rebasing error.
Please revoke approval , thanks.